packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -634,6 +636,57 @@ export const RemoteServerFactory = {
         })
     },
 
+    /**
+     * @private
+     */
+    _setupExtensions(app, options) {
+        const extensions = options.mobify?.app?.extensions || []
```

Comment:
I guess you'd have to support extension tupals here as well.

Reply:
yep, i created a [ticket](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001xW5v4YAC/view)